### DISCUSSION TITLE
multiple backup schedules with each different backup set

### DISCUSSION THREAD

**Discussion by DennisAppelman** [lavinir/hassio-onedrive-backup#234] [General]
is it possible to have for example 2 backup schedules?
for example 1 daily backup without addons and a weekly full backup?
a daily full backup uses to much data of my 4G connection…

**Comment by lavinir** [comment]
Currently this is not possible. You could as a workaround, have the second schedule backup via an automation that runs on whatever schedule you want and performs a backup as its action.
In the addon, you could enable "monitor all backups" and it will back those up to OneDrive as well.
